src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/rest-api-alerts/alerts-conditions-api-field-names.mdx

```diff
@@ -466,11 +466,11 @@ Not every field listed in this glossary is required for every condition type. Th
 
   <Collapser
     id="signal_slide_by"
-    title="signal[slideBy]"
+    title="signal[slide_by]"
   >
     By default, aggregation windows are grouped sequentially. This can lead to spiky charts whenever one window starts and another begins.
 
-    Use `slideBy` to create sliding windows. Sliding aggregated windows overlap, creating smoother charts. The `slideBy` interval sets the duration of the overlap.
+    Use `slide_by` to create sliding windows. Sliding aggregated windows overlap, creating smoother charts. The `slide_by` interval sets the duration of the overlap.
 
     In the UI, under **Advanced signal settings**, click the **Use sliding window aggregation** toggle to enable sliding windows.
 
```

Comment:
The 4th paragraph in this section says "slide by" a couple times. Should that be "slide_by" there too?